### PR TITLE
Simplify docs types (remove generics & common type aliases)

### PR DESCRIPTION
**Pull request by iMoses**

### CODE REVIEW

File: packages/ag-charts-website/src/features/api-documentation/components/FunctionCodeSample.tsx [hunk 1]
```diff
@@ -1,37 +1,48 @@
...
 import { applyInterfaceInclusions } from '../utils/applyInterfaceInclusions';
 import { escapeGenericCode, extractInterfaces, getLinkedType, writeAllInterfaces } from '../utils/documentationHelpers';
-import { getInterfaceName } from '../utils/getInterfaceName';
 import { getInterfacesToWrite } from '../utils/getInterfacesToWrite';
 import { isGridOptionEvent } from '../utils/isGridOptionEvent';
-
-export const FunctionCodeSample: FunctionComponent<FunctionCode> = ({ framework, name, type, config }) => {
+import { capitalize } from '../utils/strings';
+
+const blacklistInterfaces = [
```
Comment (alantreadway) [inline packages/ag-charts-website/src/features/api-documentation/components/FunctionCodeSample.tsx]: Nit: could we rename this to `denylistInterfaces`?

**Review by alantreadway — COMMENTED** [review]
LGTM on the `ag-charts-community` changes, I'll defer to @taktran for the website / UI changes.

**Review by taktran — APPROVED** [review]
Nice clean up work ✨ 👏🏼 👏🏼

File: packages/ag-charts-website/src/features/api-documentation/components/JsObjectPropertyView.tsx [hunk 2]
```diff
@@ -90,24 +104,20 @@ function PrimitivePropertyView({
     const framework = useContext(FrameworkContext);
     const formattedDocumentation = formatPropertyDocumentation(model).join('\n');
     const propertyType = getPropertyType(
-        (model as any).type === 'primitive' ? (model as any).tsType : model.desc.tsType
+        // model.desc.type === 'primitive' ? model.desc.aliasType ?? model.desc.tsType : model.desc.tsType
```
Comment (taktran) [inline packages/ag-charts-website/src/features/api-documentation/components/JsObjectPropertyView.tsx:107]: You can get rid of this

**Pull request merged**